src/assets/icons/Ic_Checklist.js

```diff
@@ -8,11 +8,11 @@ function Ic_Checklist(props) {
         fillRule="evenodd"
         clipRule="evenodd"
         d="M2 12c0 5.523 4.477 10 10 10s10-4.477 10-10S17.523 2 12 2 2 6.477 2 12zm2 0a8 8 0 1116 0 8 8 0 01-16 0z"
-        fill="#00ADB5"
+        fill={props.color || '#00ADB5'}
       />
       <Path
         d="M16.737 9.676a1 1 0 10-1.474-1.352l-4.727 5.157-1.768-2.121a1 1 0 10-1.536 1.28l2.5 3a1 1 0 001.505.036l5.5-6z"
-        fill="#00ADB5"
+        fill={props.color || '#00ADB5'}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good for customization. However, it's important to consider that the `props.color` might be `undefined` or an invalid color value. To ensure that a valid color is always used, you could add a default prop for `color` or validate the color before using it.
```diff
- fill={props.color || '#00ADB5'}
+ fill={isValidColor(props.color) ? props.color : '#00ADB5'}

// where isValidColor is a function that checks if the color is valid.
```
This change should be applied to both lines 11 and 15.


<!-- This is an auto-generated comment by OSS CodeRabbit -->